src/analysis/cauchy_equation.lean

```diff
@@ -0,0 +1,130 @@
+/-
+Copyright (c) 2022 Mantas Bakšys. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mantas Bakšys
+-/
+import measure_theory.measure.lebesgue.eq_haar
+
+/-!
+# Cauchy's Functional Equation
+
+This file contains the classical results about the Cauchy's functional equation
+`f (x + y) = f x + f y` for functions `f : ℝ → ℝ`. In this file, we prove that the solutions to this
+equation are linear up to the case when `f` is a Lebesgue measurable functions, while also deducing
+intermediate well-known variants.
+-/
+
+open add_monoid_hom measure_theory measure_theory.measure metric nnreal set
+open_locale pointwise topology
+
+section seminormed_group
+open topological_space
+variables {G H : Type*} [seminormed_add_group G] [topological_add_group G] [is_R_or_C H] {s : set G}
+
+lemma add_monoid_hom.continuous_of_bounded_nhds_zero (f : G →+ H) (hs : s ∈ 𝓝 (0 : G))
```

Comment:
Isn't `[is_R_or_C H]` too strong? Is it true for any normed space over rationals?